### ISSUE TITLE
Dashed Curved Lines

### ISSUE THREAD

**Issue by henryjsneed** [davidfig/pixi-dashed-line#5]
Would it be possible to add support for the bezierTo function so that we could create dashed curved lines? Are there any fundamental limitations to doing this?

**Comment by davidfig** [comment]
Yes, definitely possible. Need to copy the bezierTo code from pixi into the library and use the same logic as the rest of the statements. Happy to accept a PR.

**Comment by jtwaleson** [comment]
Implemented here: https://github.com/jtwaleson/pixi-dashed-line/tree/bezier
